suplemon/main.py

```diff
@@ -182,14 +190,14 @@ def main_loop(self):
 
                 if event:
                     got_input = True
-                    self.on_input(event)
+                    self.on_input(event)  # Up to 30% processing time
```

Comment:
Ah, k. It might be wise to add a `PERF:` prefix to the comment so it can easily be found via `git grep` during cleanup